projects/hslayers/src/components/add-data/catalogue/add-data-catalogue.component.ts

```diff
@@ -53,7 +53,7 @@ export class HsAddDataCatalogueComponent {
     this.loaderImage =
       this.HsUtilsService.getAssetsPath() + 'img/ajax-loader.gif';
 
-    this.reload();
+    console.log('ts')
```

Comment:
Whats this? :dancers:

Reply:
Some leftover console log as pre usual :D . Reloads supposed to be removed